testing/data/DATA_LICENSES.md

```diff
@@ -39,6 +39,7 @@
 - vase_4comp.vti: VTK Data: BSD-3-Clause
 - WaterBottle.glb: glTF-Sample-Models: Public Domain
 - world*: VTK Data: BSD-3-Clause
+- 16bits.*: @bisechen: Public Domain
```

Comment:
@bisechen I've put a "Public Domain" license. Let me know if it works for you or if you prefer another license.

Reply:
CC0 would be better than public domain if choice is possible

Reply:
Ok. Please @bisechen let us know if you are ok with cc0

Reply:
> 還行。如果您對cc0沒問題，請告訴我們

This is only a temporary use to test the 16 and 32 bit color depth file of F3D 2.0, not a "work", so CC0 is not a problem. ^^

Reply:
We are trying to be respectful of other people's work, even if it's a quick test model. Thank you for accepting the CC0 license :)